xbmc/pvr/recordings/PVRRecording.cpp

```diff
@@ -91,6 +91,7 @@ CPVRRecording::CPVRRecording(const PVR_RECORDING &recording, unsigned int iClien
   m_bIsDeleted                     = recording.bIsDeleted;
   m_iEpgEventId                    = recording.iEpgEventId;
   m_iChannelUid                    = recording.iChannelUid;
+  m_iRecordingState                = recording.iRecordingState;
```

Comment:
assert() if value is PVR_RECORDED_TV_ITEM_STATE_INVALID! We must force the addon devs to support this property.